customize.dist/src/less2/pages/page-features.less

```diff
@@ -112,6 +112,7 @@
     .cp-regis-user {
         @media (max-width:575px) {
             margin-top: 3em;
+            margin-bottom: 3em;
```

Comment:
I recommend adding an extra `margin-top` to the 3rd card, if it exists, instead of a `margin-bottom` here.
The 3rd card is only visible on cryptpad.fr where premium subscription are available so here you're adding an unnecessary margin for all the other instances.